src/components/MovieDetails/Component/MovieDetails.tsx

```diff
@@ -0,0 +1,36 @@
+import { Movie } from "../../Types/movie";
+import styles from "../Styles/MovieDetails.module.css";
+
+interface Props {
+  movie: Movie;
+}
+
+export default function MovieDetails({ movie }: Props) {
```

Comment:
You are passing into the MovieDetails from the App the object that is not a Movie type. As a result - there is no duration in UI. TS is intended to prevent this case.

![Screenshot 2024-03-05 185214](https://github.com/keber23/react-learn/assets/79011727/327c4f26-44fa-4aa9-95ae-cd7df31bb95a)